src/main/java/core/basesyntax/Lottery.java

```diff
@@ -0,0 +1,23 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class Lottery {
+    public static Ball getRandomBall() {
+        Color randomColor = ColorSupplier.getRandomColor();
+        Random random = new Random();
+        int randomNumber = random.nextInt(100) + 1;
```

Comment:
100 its magic number, move to the constant
an example can be seen in the checklist

Reply:
not fixed